docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -8,6 +8,26 @@ ifdef::notBuildingForSite[]
 include::{includedir}/menu.adoc[]
 endif::notBuildingForSite[]
 
+== Getting Ehcache
+
+From Maven
+
+    <dependencies>
+        <dependency>
+          <groupId>org.ehcache</groupId>
+          <artifactId>ehcache</artifactId>
+          <version>3.10.0</version>
+        </dependency>
+    </dependencies>
+
+From Gradle
+
+    org.ehcache:ehcache:3.10.0
+
+ehcache-xml and ehcache modules requires jaxb dependency at runtime, So If you are using java 1.9+ version then you should make jaxb dependency available at runtime.
```

Comment:
I wouldn't bother to reference `ehcache-xml` here since you haven't mentioned it the the snippets above.

I'm not sure the emphasis is correct here either. Since the maven dependency is now `runtime` scoped for most users inclusion of JAXB on the classpath will be automatic. There are however two things that may be worth pointing out:
1. We depend on a version range. This means unless the user picks a version the build will be unstable.
2. If they want to use Jakarta EE artifacts they will need to select the Jakarta variants using the appropriate Maven/Gradle invocation.